src/commons/crypto/signing/signers/kmip/internal.rs

```diff
@@ -190,7 +202,11 @@ impl KmipSigner {
         // Signer initialization should not block Krill startup. As such we delaying contacting the KMIP server until
         // first use. The downside of this approach is that we won't detect any issues until that point.
 
-        let server = Arc::new(StatefulProbe::new(Arc::new(conf.try_into()?), Duration::from_secs(30)));
+        let server = Arc::new(StatefulProbe::new(
+            name.to_string(),
+            Arc::new(conf.try_into()?),
+            Duration::from_secs(30),
```

Comment:
This makes me wonder whether the `probe_interval` should be part of the `KmipSignerConfig`. We can always do this later of course..

Reply:
I linked to this comment from https://github.com/NLnetLabs/krill/issues/726.